autonomy/cli/utils/spec_helpers.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""FSM spec helpers."""
+
+import importlib
+from pathlib import Path
+from types import ModuleType
+from typing import Optional, cast
+
+import click
+
+from autonomy.analyse.abci.app_spec import (
+    DFA,
+    DFASpecificationError,
+    FSMSpecificationLoader,
+    check_unreferenced_events,
+)
+
+
+def import_and_validate_app_class(module_path: Path, app_class: str) -> ModuleType:
+    """Import and validate rounds.py module."""
+
+    module_to_look_for = "rounds"
+    if not (module_path / "rounds.py").exists():
+        module_to_look_for = "composition"
```

Comment:
@Adamantios In the register reset abci, the ABCI app is defined in `compositions.py` and other has it in `rounds.py`, is there a specific rule to it if not we need one

Reply:
We can make the CLI agnostic but we need some rules to implement the logic